lib/utils/object.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Aliasing Object[method] allows the minifier to shorten methods to a single character variable,
+ * as well as giving BV a chance to inject polyfills.
+ * As long as we avoid
+ * - import * as Object from "utils/object"
+ * all unused exports should be removed by tree-shaking.
+ */
+
+export const assign = Object.assign
```

Comment:
the `Object.assign` polyfill is quite short and could be assigned to `assign` if no native `Object.assign`